extensions/s3/src/main/java/io/deephaven/extensions/s3/GCSSeekableChannelProviderPlugin.java

```diff
@@ -25,14 +25,14 @@ public final class GCSSeekableChannelProviderPlugin implements SeekableChannelsP
             S3Instructions.builder().endpointOverride(DEFAULT_ENDPOINT_OVERRIDE).build();
 
     @Override
-    public boolean isCompatible(@NotNull final URI uri, @Nullable final Object config) {
-        return GCS_URI_SCHEME.equals(uri.getScheme());
+    public boolean isCompatible(@NotNull String uriScheme, @Nullable Object config) {
```

Comment:
Should match the S3 impl, even though it doesn't matter.
```suggestion
    public boolean isCompatible(@NotNull final String uriScheme, @Nullable final Object config) {
```